mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -113,3 +113,9 @@ Description of the change and how users should adjust their configuration (if ne
 ### `deletion_policy` now defaults to `DELETE`
 
 Previously, `google_firebase_web_app` deletions default to `ABANDON`, which means to only stop tracking the WebApp in Terraform. The actual app is not deleted from the Firebase project. If you are relying on this behavior, set `deletion_policy` to `ABANDON` explicitly in the new version.
+
+## Resource: `google_identity_platform_project_default_config`
+
+### Deprecated
+
+Use the `google_identity_platform_config` resource instead.
```

Comment:
```suggestion
### `google_identity_platform_project_default_config` has been removed from the provider

Use the `google_identity_platform_config` resource instead.
```
Using https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/version_3_upgrade#google_project_services-has-been-removed-from-the-provider as an example. We may want to give a few more details on why it was removed